tests/Unit/ProtoTests/BasicOneof/out/src/Request/SupplementaryDataOneof.php

```diff
@@ -67,4 +67,207 @@ class SupplementaryDataOneof
      * lower_camel_case.
      */
     private static $selectedOneofFieldName = '';
+
+    /**
+     * Sets this oneof to extra_description and updates its value.
+     *
+     * @param mixed $extraDescription The new value of this oneof.
+     */
+    public function setExtraDescription($extraDescription)
+    {
+        $this->extraDescription = $extraDescription;
+        $this->selectedOneofFieldName = extra_description;
```

Comment:
Should this be `'extra_description'`/`"extra_description"`? Or is this a variable defined somewhere that I'm missing?

Same for the rest.